examples/tutorials/advanced/websockets-example-MNIST-parallel/start_websocket_servers.py

```diff
@@ -17,49 +18,67 @@
     ),
 )
 
+
+if os.name == "nt":
+    python = "python"
+else:
+    python = "python" + sys.version[0:3]
+
+FILE_PATH = Path(os.path.abspath(__file__)).parent.parent.parent.parent.parent.joinpath(
```

Comment:
```suggestion
FILE_PATH = Path(__file__).resolve().parents[4].joinpath(
```
Should work. This way we don't need to get `os` involved

Reply:
Done